packages/glibc_standalone.rb

```diff
@@ -0,0 +1,101 @@
+require 'package'
+
+class Glibc_standalone < Package
+  description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
+  homepage 'https://www.gnu.org/software/libc/'
+  version '2.41'
+  license 'LGPL-2.1+, BSD, HPND, ISC, inner-net, rc, and PCRE'
+  compatibility 'aarch64 armv7l x86_64'
+  source_url "https://ftpmirror.gnu.org/glibc/glibc-#{version}.tar.xz"
+  source_sha256 'a5a26b22f545d6b7d7b3dd828e11e428f24f4fac43c934fb071b6a7d0828e901'
+  binary_compression 'tar.zst'
+
+  depends_on 'gawk' => :build
+  depends_on 'filecmd' # L Fixes creating symlinks on a fresh install.
+  depends_on 'libidn2' => :build
+  depends_on 'llvm' => :build
+  depends_on 'texinfo' => :build
+  depends_on 'libxcrypt' # Latest glibc removed libcrypt.so, add this for backward compatibility
+  depends_on 'patchelf' # L
+
+  conflicts_ok
+  no_env_options
+  no_shrink
+
+  def self.patch
+    File.write 'hardcode-library-path.patch', <<~EOF
+      diff -Nur a/elf/dl-support.c b/elf/dl-support.c
+      --- a/elf/dl-support.c    2025-04-23 01:58:17.265974714 +0800
+      +++ b/elf/dl-support.c    2025-04-23 14:38:27.373033131 +0800
+      @@ -296,7 +296,7 @@
+
+         /* Initialize the data structures for the search paths for shared
+            objects.  */
+      -  _dl_init_paths (getenv ("LD_LIBRARY_PATH"), "LD_LIBRARY_PATH",
+      +  _dl_init_paths ("#{CREW_PREFIX}/opt/glibc-libs:#{CREW_LIB_PREFIX}:/#{ARCH_LIB}", "LD_LIBRARY_PATH",
```

Comment:
Why `#{CREW_LIB_PREFIX}:/#{ARCH_LIB}`? Isn't that redundant?

Reply:
It is ChromeOS's default `LD_LIBRARY_PATH`:
```
$ env | grep LD_LIBRARY
LD_LIBRARY_PATH=/usr/local/lib64:/lib64
```

Reply:
Oops. I misread that.